src/routes/blog/[[page]]/+page.svelte

```diff
@@ -223,14 +223,14 @@
                             )}
                         {@const featuredAuthors = featuredAuthorSlugs
                             .map((slug) => data.authors.find((a) => a.slug === slug))
-                            .filter((a): a is NonNullable<typeof a> => Boolean(a))}
```

Comment:
The `prettier format` command removes this `<typeof a>` but removing it is causing the diagnostics test to fail